.github/workflows/php.yml

```diff
@@ -45,3 +46,4 @@ jobs:
           files: "**.php" # you may customize glob as needed
           phpcs_path: php phpcs.phar
           standard: phpcs.xml
+          scope: "file"
```

Comment:
There's a bug that causes the default to not respect the standards xml: https://github.com/tinovyatkin/action-php-codesniffer/issues/40